### ISSUE TITLE
Il-2 Sturmovik: 1946 (15320)

### ISSUE THREAD

**Issue by Talkless** [ValveSoftware/Proton#2424]
# Compatibility Report
- Name of the game with compatibility issues: Il-2 Sturvmovik: 1946
- Steam AppID of the game: 15320

## System Information
- GPU: NVIDIA Corporation GM107M [GeForce GTX 860M] 
- Driver/LLVM version: nvidia-legacy-390xx-driver 390.116-1
- Kernel version: 4.19.28-2 (2019-03-15)
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Talkless/f63cdef47813f13712987d2b9d28b4f7
- Proton version: 3.16-8 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-15320.log](https://github.com/ValveSoftware/Proton/files/2974206/steam-15320.log)

## Symptoms 

Game stopped working after some recent Debian Sid ugprades. Other native game SOMA works OK, and Il-2 worked OK too some time ago. Now it just doesn't start. Console shows this:

```
 *** Library **** DT ****  Loaded *** 102
RTS Version 2.2
Core Version 2.0
Sound: Native library (build 1.1, target - P IV) loaded.
Main begin: ScreenMode NOT changed
Main end: null
```
Basically, game window does not appear.

No difference if I start using "optirun" to enable discrete NVIDIA card (as I always do), or without (using Intel graphics).

## Reproduction

Just click Play in Steam app.

**Comment by Talkless** [comment]
Interesting snippet from steam log:
```
4062.791:0008:0009:trace:module:load_dll Found L"C:\\windows\\system32\\opengl32.dll" for L"Opengl32.dll" at 0x7a820000, count=2
4062.791:0008:0009:trace:module:GetModuleFileNameW L"C:\\windows\\system32\\opengl32.dll"
OpenGL provider: Opengl32.dll
com.maddox.opengl.GLContextException: ScreenMode NOT changed
	at com.maddox.il2.engine.Config.createGlContext(Config.java:542)
	at com.maddox.il2.engine.Config.createGlContext(Config.java:595)
	at com.maddox.il2.game.Main3D.beginApp(Main3D.java:507)
	at com.maddox.il2.game.MainWin3D.beginApp(MainWin3D.java:212)
	at com.maddox.il2.game.Main.exec(Main.java:422)
	at com.maddox.il2.game.GameWin3D.main(GameWin3D.java:235)
java.lang.NullPointerException
	at com.maddox.rts.CfgTools.save(CfgTools.java:59)
	at com.maddox.sound.AudioDevice.saveControls(AudioDevice.java:385)
	at com.maddox.il2.engine.Config.saveSound(Config.java:484)
	at com.maddox.il2.engine.Config.save(Config.java:299)
	at com.maddox.il2.game.MainWin3D.endApp(MainWin3D.java:168)
	at com.maddox.il2.game.Main.exec(Main.java:451)
	at com.maddox.il2.game.GameWin3D.main(GameWin3D.java:235)
```

**Comment by Talkless** [comment]
I've fixed it by uninstalling Il-2, *and* manually removing leftover directory `.steam/steam/steamapps/common/IL 2 Sturmovik 1946`. It launches now!

Maybe I've changed some settings in the way it made it to break?

**Comment by Talkless** [comment]
I believe I've accidentally selected 1920x1080**x16** mode instead of 1920x1080**x32**, as I managed to break it again... So it's not the Proton issue.
